src/App/Order/Model/PdkOrder.php

```diff
@@ -149,6 +154,7 @@ class PdkOrder extends Model
     public function __construct(?array $data = null)
     {
         parent::__construct($data);
+        $this->initializeHasAudits();
```

Comment:
dit moet in model verwerkt worden, op de manier die ik je heb laten zien. dit kan je niet handmatig gaan aanroepen, is te foutgevoelig